src/main/java/cms/gov/madie/measure/services/GroupService.java

```diff
@@ -308,9 +309,49 @@ private TestCaseStratificationValue updateTestCaseStratification(
               .build();
     }
     testCaseStrata.setName(strataName);
+    handlePopulationChange(testCaseStrata, testCaseGroup);
+
     return testCaseStrata;
   }
 
+  private void handlePopulationChange(
```

Comment:
Ok I may not be understanding this, but can we do ? as far as I understand we are trying to make sure that populations that are added in MeasureGroups are updated in Test Cases. If that is the case, then why can't we just copy them over instead of manually verifying the removed groups or manually adding new groups ?

`testCaseStrata.setPopulationValues(testCasePopulationValuesFromGroup);`